docs/config-app.md

```diff
@@ -229,7 +229,10 @@ If not defined in config all other Health Checkers would be disabled and endpoin
 - `gdpr.vendorlist.http-endpoint-template` - template string for vendor list url, where `{VERSION}` is used as version number placeholder.
 - `gdpr.vendorlist.http-default-timeout-ms` - default operation timeout for obtaining new vendor list.
 - `gdpr.vendorlist.filesystem-cache-dir` - directory for local storage cache for vendor list. Should be with `WRITE` permissions for user application run from.
-- `gdpr.geolocation.enabled` - if equals to `true` the geo location service will be used to determine the country for client request.
+
+## Geo Location
```

Comment:
Here is the proposal for contents of this section:
```
## Geo Location
- `geolocation.enabled` - if equals to `true` the geo location service will be used to determine the country for client request.
- `geolocation.circuit-breaker.enabled` - if equals to `true` circuit breaker will be used to make geo location client more robust.
- `geolocation.circuit-breaker.opening-threshold` - the number of failures before opening the circuit.
- `geolocation.circuit-breaker.opening-interval-ms` - time interval for opening the circuit breaker if failures count reached.
- `geolocation.circuit-breaker.closing-interval-ms` - time spent in open state before attempting to re-try.
- `geolocation.type` - set the geo location service provider, can be `maxmind` or custom provided by hosting company.
- `geolocation.maxmind` - section for [MaxMind](https://www.maxmind.com) configuration as geo location service provider.
- `geolocation.maxmind.remote-file-syncer` - use RemoteFileSyncer component for downloading/updating MaxMind database file.
- `geolocation.maxmind.remote-file-syncer.download-url` - url to database file to download.
- `geolocation.maxmind.remote-file-syncer.save-filepath` - local path to downloaded database file.
- `geolocation.maxmind.remote-file-syncer.retry-count` - how many times try to download.
- `geolocation.maxmind.remote-file-syncer.retry-interval-ms` - how long to wait between failed retries.
- `geolocation.maxmind.remote-file-syncer.timeout-ms` - default operation timeout for obtaining database file.
- `geolocation.maxmind.remote-file-syncer.http-client.connect-timeout-ms` - set the connect timeout.
- `geolocation.maxmind.remote-file-syncer.http-client.max-redirects` - set the maximum amount of HTTP redirections to follow. A value of 0 (the default) prevents redirections from being followed.
```

Reply:
Will do, thanks! Although, we already have a `remote-file-syncer` properties described in a separate category like `<SERVICE>.remote-file-syncer.*` so should we add basically a duplicate for `maxmind`?

Reply:
In this case no duplication is necessary but it would be good to leave `geolocation.maxmind.remote-file-syncer` group description with a reference to `## Remote-file-syncer` section. What do you think?

Reply:
Definitely think that's a better thing to do. We could apply the same logic to `<service>.circuit-breaker.*` as we currently have 3 pairs of those.